react/src/pages/RoutingListPage.tsx

```diff
@@ -201,6 +203,38 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
     return color;
   };
 
+  const sessionOwnerElement = React.useMemo(() => {
+    if (!baiClient.supports('model-serving-endpoint-user-info'))
+      return baiClient.email || '';
+    if (endpoint?.created_user_email === endpoint?.session_owner_email)
+      return endpoint?.session_owner_email || '';
+    else
+      return (
+        <>
+          {endpoint?.session_owner_email || ''}
+          <Tooltip
+            title={t('modelService.ServiceDelegatedFrom', {
+              createdUser: endpoint?.created_user_email || '',
+              sessionOwner: endpoint?.session_owner_email || '',
+            })}
+          >
+            <Button
+              size="small"
+              type="text"
+              icon={<QuestionCircleOutlined />}
+              style={{ color: token.colorTextSecondary }}
+            />
+          </Tooltip>
+        </>
+      );
+  }, [
+    endpoint?.created_user_email,
+    endpoint?.session_owner_email,
+    baiClient,
+    t,
+    token.colorTextSecondary,
+  ]);
+
```

Comment:
Let's make this part as an individual component (eg. `<ServiceSessionOwner serviceFrgmt={service}/>`) using a GraphQL fragment. After that, we can reuse this component in [`ServingListPage.tsx`](https://github.com/lablup/backend.ai-webui/blob/cbc4bde074ed8f6562281af044a558eb0eddf0b7/react/src/pages/ServingListPage.tsx#L176-L185)

You can refer [`EndpointStatusTag.tsx`](https://github.com/lablup/backend.ai-webui/blob/cbc4bde074ed8f6562281af044a558eb0eddf0b7/react/src/components/EndpointStatusTag.tsx#L13C1-L20) which uses GraphQL fragment.

Reply:
Oops. I selected wrong line. The code what I want to mention is [this](https://github.com/lablup/backend.ai-webui/pull/2138/files/cbc4bde074ed8f6562281af044a558eb0eddf0b7#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8R206-R236)